docker-entrypoint.sh

```diff
@@ -14,6 +14,12 @@ if [[ $DOCKER_HOST == unix://* ]]; then
 	fi
 fi
 
+# Compute the DNS resolvers for use in the templates
+export RESOLVERS=$(awk '$1 == "nameserver" {print $2}' ORS=' ' /etc/resolv.conf | sed 's/ *$//g')
```

Comment:
There unnecessary use of cat and grep (since they already need awk) suggest that my method is better.  I also trim leading whitespace.  Also,  by blindly using grep, they would also include nameservers that are commented out or any other lines that happen to contain "nameserver".